index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse Pair Programming Tribute Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="main">
+      <!-- header -->
+      <div class="header-container myflex">
+        <div class="head-content">
+          <h1 id="title">President Buhari</h1>
+        </div>
+      </div>
```

Comment:
- [ ] [OPTIONAL] Consider using a `header` tag as it is the appropriate semantic tag for headers display